### PR TITLE
Add team page

### PR DESCRIPTION
**Pull request by dburkhardt**

### CODE REVIEW

File: config/_default/menus.yaml [hunk 1]
```diff
@@ -28,3 +28,6 @@ main:
   - name: Join
     url: /#join-us
     weight: 100
+  - name: Team
```
Comment (scottgigante) [inline config/_default/menus.yaml:31]: I would put team before join

**Review by scottgigante — COMMENTED** [review]
It's a bit weird that team members are listed alphabetical by first name, but it's a bit unclear to me if it's possible to fix that with this widget

File: content/authors/jonathan_b/_index.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
+---
+# Display name
+title: Jonathan Bloom, PhD
+
+# Is this the primary user of the site?
+superuser: false
+
+# Role/position
+role: Purposefully Unemployed; Formerly Cellarity and Broad Institute
```
Comment (jbloom22) [inline content/authors/jonathan_b/_index.md:9]: Purposefully Unemployed; formerly Cellarity, Broad Institute, MIT

File: content/authors/jonathan_b/_index.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+  - name: ''
+    url: ''
+
+# Short bio (displayed in user profile at end of posts)
+bio: 
+
+social:
+  - icon: envelope
+    icon_pack: fas
+    link: 'mailto:johnny@appleseed.com'
```
Comment (jbloom22) [inline content/authors/jonathan_b/_index.md:22]: jonathan.bloom.cv@gmail.com

File: content/authors/jonathan_b/_index.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+  - icon: twitter
+    icon_pack: fab
+    link: https://twitter.com/johnnyappleseed
+  - icon: github
+    icon_pack: fab
+    link: https://github.com/johnnyappleseed
+  - icon: linkedin
+    icon_pack: fab
+    link: https://www.linkedin.com/in/johnny-appleseed/
```
Comment (jbloom22) [inline content/authors/jonathan_b/_index.md:31]: https://www.linkedin.com/in/jon-bloom/

File: content/authors/jonathan_b/_index.md [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+social:
+  - icon: envelope
+    icon_pack: fas
+    link: 'mailto:johnny@appleseed.com'
+  - icon: twitter
+    icon_pack: fab
+    link: https://twitter.com/johnnyappleseed
+  - icon: github
+    icon_pack: fab
+    link: https://github.com/johnnyappleseed
```
Comment (jbloom22) [inline content/authors/jonathan_b/_index.md:28]: https://github.com/jbloom22

File: content/authors/jonathan_b/_index.md [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+# Short bio (displayed in user profile at end of posts)
+bio: 
+
+social:
+  - icon: envelope
+    icon_pack: fas
+    link: 'mailto:johnny@appleseed.com'
+  - icon: twitter
+    icon_pack: fab
+    link: https://twitter.com/johnnyappleseed
```
Comment (jbloom22) [inline content/authors/jonathan_b/_index.md:25]: https://twitter.com/MorseCell

File: content/authors/jonathan_b/_index.md [hunk 7]
```diff
@@ -0,0 +1,44 @@
+---
+# Display name
+title: Jonathan Bloom, PhD
```
Comment (jbloom22) [inline content/authors/jonathan_b/_index.md:3]: Jonathan M. Bloom, PhD

**Review by jbloom22 — CHANGES_REQUESTED** [review]
I don't think I'm authorized to PR, edits in comments below.

**Pull request merged**